src/components/Avatar/__stories__/avatar.stories.mdx

```diff
@@ -120,92 +143,131 @@ Use for non-person avatars, such as a workspace or team.
+          </div>
+        ),
+        description: "Avoid using a mix of avatar sizes that display together and create design imbalance."
       }
     },
     {
       positive: {
-        component:<Avatar src={maleIcon} type={Avatar.types.IMG} />,
-        description:"Use branded generic avatars when a user has not set their avatar image."
+        component: <Avatar src={maleIcon} type={Avatar.types.IMG} />,
+        description: "Use branded generic avatars when a user has not set their avatar image."
       },
       negative: {
-        component:<Avatar src={femaleIcon} type={Avatar.types.IMG} />,
-        description:"Don’t make assumptions and use gendered placeholder avatars."
+        component: <Avatar src={femaleIcon} type={Avatar.types.IMG} />,
+        description: "Don’t make assumptions and use gendered placeholder avatars."
       }
     }
   ]}
 />
 
 ## Use cases and examples
+
 ### Multiple avatars
+
 When there is more than one avatar, use an -8 margin between avatars.
+
 <Canvas>
-  <Story name="Multiple avatars" >
+  <Story name="Multiple avatars">
     <MultipleStoryElementsWrapper className="monday-style-story-avatar_multiple">
-      <Avatar size={Avatar.sizes.LARGE} type={Avatar.types.IMG} src={person1}  />
+      <Avatar size={Avatar.sizes.LARGE} type={Avatar.types.IMG} src={person1} />
       <Avatar size={Avatar.sizes.LARGE} type={Avatar.types.IMG} src={person2} />
       <Avatar size={Avatar.sizes.LARGE} type={Avatar.types.IMG} src={person3} />
-  </MultipleStoryElementsWrapper>
+    </MultipleStoryElementsWrapper>
   </Story>
 </Canvas>
 
 ### Avatar with right badge
+
 Use to indicate the user’s permissions such as: Guest, owner.
 
 <Canvas>
   <Story name="Avatar with right badge">
-    <Avatar
-        size={Avatar.sizes.LARGE} 
-        type={Avatar.types.IMG} 
-        src={person1} 
-        bottomRightBadgeProps={{src: guest}}
-    />
-    <Avatar
-        size={Avatar.sizes.LARGE} 
-        type={Avatar.types.IMG} 
-        src={person1} 
-        bottomRightBadgeProps={{src: owner}}
-    />
+    <Avatar size={Avatar.sizes.LARGE} type={Avatar.types.IMG} src={person1} bottomRightBadgeProps={{ src: guest }} />
+    <Avatar size={Avatar.sizes.LARGE} type={Avatar.types.IMG} src={person1} bottomRightBadgeProps={{ src: owner }} />
   </Story>
-</Canvas> 
+</Canvas>
 
 ### Avatar with left badge
+
 Use to indicate the status of a user such as: Working from home, out of office etc.
+
 <Canvas>
   <Story name="Avatar with left badge">
-    <Avatar 
-        size={Avatar.sizes.LARGE} 
-        type={Avatar.types.IMG} 
-        src={person1} 
-        bottomLeftBadgeProps={{src: home}} 
+    <Avatar size={Avatar.sizes.LARGE} type={Avatar.types.IMG} src={person1} bottomLeftBadgeProps={{ src: home }} />
+    <Avatar size={Avatar.sizes.LARGE} type={Avatar.types.IMG} src={person1} bottomLeftBadgeProps={{ src: minus }} />
+  </Story>
+</Canvas>
+
+### Avatar with tooltip
```

Comment:
please add here another example with aria label as tooltip content

and write explanation above each avatar with tooltip:
Avatar with textual representation as tooltip content
and Avatar with tooltip content which is different from it's textual representation

and please add aria label to both examples (because we do want developers always add aria label to their avatars)

Reply:
Not 100% clear, let's discuss